src/components/v5/common/CompletedAction/partials/SelectedMembers/partials/MembersTable/MembersTable.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useMemo } from 'react';
+
+import Table from '~v5/common/Table/Table.tsx';
+
+import { type SelectedMember } from '../../types.ts';
+
+import { type MembersTableModel, membersColumns } from './tableFixtures.tsx';
+
+interface MembersTableProps {
+  members: SelectedMember[];
+}
+
+const displayName =
+  'v5.common.CompletedAction.partials.SelectedMembers.partials.MembersTable';
+
+const MembersTable = ({ members }: MembersTableProps) => {
+  const columns = useMemo(() => membersColumns, []);
```

Comment:
it used to be so you had to memo the columns, but this was a previous version, lemme go over the docs and I'll get back to you :D

Reply:
https://tanstack.com/table/v8/docs/guide/migrating#update-column-definitions yeah if you define it inside a component, it's recommended to wrap them inside of `useMemo`, but they're in a separate file here, so I can remove this one. I guess this would be the way to do it if we pass some values from the component into the column helper